apps/ledger-live-mobile/e2e/specs/swap/swap.spec.ts

```diff
@@ -43,10 +43,11 @@ describe("Swap", () => {
     await swapPage.selectAccount("Ethereum");
   });
 
-  it("should be able to send the maximum available amount", async () => {
-    await swapPage.sendMax();
-    await swapPage.startExchange();
-    await expect(swapPage.termsAcceptButton()).toBeVisible();
-    await expect(swapPage.termsCloseButton()).toBeVisible();
-  });
+  // TODO: Re-enable once max related bugs have been resolved.
+  // it("should be able to send the maximum available amount", async () => {
+  //   await swapPage.sendMax();
+  //   await swapPage.startExchange();
+  //   await expect(swapPage.termsAcceptButton()).toBeVisible();
+  //   await expect(swapPage.termsCloseButton()).toBeVisible();
```

Comment:
add another test case similar to this one, replacing swapPage.sendMax() by swapPage.enterSourceAmount("0.05");
I'm not sure of the 0.05, depends on what's available in this fake account.